File: week1/mario:more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Prompt user for a positive integer ONLY between 1 and 8
    int i;
    do
    {
        i = get_int("Choose a number between 1 and 8: ");
    }
    while (i > 8 || i < 1);

    //Print pyramid
    int l = 1;
    while (i > 0)
    {
        //Print space(s)
        int s = i - 1;
        do
        {
            if (s != 0)
            {
                printf(" ");
            }
            s--;
        }
        while (s > 0);

        //Print left hash(es)
        int lh = 0;
        do
        {
            printf("#");
            lh++;
        }
        while (lh < l);

        //Print middle hashes
        printf("  ");

        //Print right hash(es)
        int rh = 0;
        do
        {
            printf("#");
            rh++;
        }
        while (rh < l);

        //Print new line
        printf("\n");
        l++;
        i--;
    }
}

File: week6/sentimental:mario:more/mario.py
# TODO
# Prompt user for a height (integer)
# Print a pyramid of '#' with corresponding height


def main():
    height = get_height()
    # Print left hashes
    for i in range(height):
        i += 1
        print(" " * (height - i), end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)


def get_height():
    while (True):
        try:
            i = int(input("Height: "))
            if i > 0 and i < 9:
                return i
        except ValueError:
            print("Not an integer.")


main()